src/cascadia/WindowsTerminal/WindowsTerminal.vcxproj

```diff
@@ -90,6 +90,16 @@
       <DeploymentContent>true</DeploymentContent>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </NativeReferenceFile>
+
```

Comment:
What happened here? Does this need to be added again? Is github acting weird?

Reply:
I don't know what happened here, I did a git rebase to make sure.
I am investigating